src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -112,10 +118,10 @@ internal static string GetCacheFilePath(RestoreRequest request, LockFile lockFil
         /// When the project has opted into packages lock file, it also verified that the lock file is present on disk.
         /// This does not account if the files were manually modified since the last restore
         /// </summary>
-        internal static bool VerifyRestoreOutput(RestoreRequest request)
+        internal static bool VerifyRestoreOutput(RestoreRequest request, CacheFile cacheFile)
         {
 
-            if (!File.Exists(request.ExistingLockFile?.Path))
+            if (!File.Exists(request.LockFilePath))
```

Comment:
Is this set in the dotnet cli tool case? 

Also can you please remove the extra line.

Reply:
Looks like if no tool is found then this can be null, I'll add a null check